docs/how-to/plan-upgrade.rst

```diff
@@ -23,72 +27,118 @@ Usage example
-        Upgrade plan for 'neutron-openvswitch' to 'victoria'
-            Upgrade 'neutron-openvswitch' to the new channel: 'victoria/stable'
+            Upgrade plan for 'keystone-ldap' to 'victoria'
+                Refresh 'keystone-ldap' to the latest revision of 'ussuri/stable'
+                Upgrade 'keystone-ldap' to the new channel: 'victoria/stable'
+        Upgrading all applications deployed on machines with hypervisor.
+            Upgrade plan for 'az-1' to 'victoria'
+                Upgrade software packages of 'nova-compute' from the current APT repositories
+                    Upgrade software packages on unit 'nova-compute/0'
+                Refresh 'nova-compute' to the latest revision of 'ussuri/stable'
+                Change charm config of 'nova-compute' 'action-managed-upgrade' to True
+                Upgrade 'nova-compute' to the new channel: 'victoria/stable'
+                Change charm config of 'nova-compute' 'source' to 'cloud:focal-victoria'
+                Upgrade plan for units: nova-compute/0
+                    Upgrade plan for unit 'nova-compute/0'
+                        Disable nova-compute scheduler from unit: 'nova-compute/0'
+                        Verify that unit 'nova-compute/0' has no VMs running
+                        ├── Pause the unit: 'nova-compute/0'
+                        ├── Upgrade the unit: 'nova-compute/0'
+                        ├── Resume the unit: 'nova-compute/0'
+                        Enable nova-compute scheduler from unit: 'nova-compute/0'
+                Wait for up to 1800s for model 'test_model' to reach the idle state
+                Verify that the workload of 'nova-compute' has been upgraded on units: nova-compute/0
+        Remaining Data Plane principal(s) upgrade plan
+            Upgrade plan for 'ceph-osd' to 'victoria'
+                Verify that all 'nova-compute' units had been upgraded
+                Upgrade software packages of 'ceph-osd' from the current APT repositories
+                    Upgrade software packages on unit 'ceph-osd/0'
+                Change charm config of 'ceph-osd' 'source' to 'cloud:focal-victoria'
+                Wait for up to 300s for app 'ceph-osd' to reach the idle state
+                Verify that the workload of 'ceph-osd' has been upgraded on units: ceph-osd/0
+        Data Plane subordinate(s) upgrade plan
+            Upgrade plan for 'ovn-chassis' to 'victoria'
+                Refresh 'ovn-chassis' to the latest revision of '22.03/stable'
+
+Plan for the control-plane
+--------------------------
+
+To generate a plan targeting the **control-plane** applications use:
+
+.. code:: bash
+
+    cou plan control-plane
+
+Plan for the data-plane
+-----------------------
+
+To generate a plan targeting the **data-plane** applications use:
+
+.. code:: bash
+
+    cou plan data-plane
+
+**Note:**
+- It's essential to complete the upgrade of the **control-plane** components before being able to
```

Comment:
An empty line is needed before the list items, otherwise they are not correctly formatted. There are a few places that also need to have this changed.